net/zmap/Portfile

```
@@ -16,25 +16,30 @@ long_description    designed for Internet-wide network surveys capable of \
                     a 10GbE connection.
 homepage            https://zmap.io/
```

Please use `github.tarball_from archive` rather than the default and deprecated value of `tarball`. When you make this change, will need to update the checksums as well.
```suggestion
homepage            https://zmap.io/
github.tarball_from archive
```